test_websocket_with_instructions.py

```
@@ -0,0 +1,352 @@
+        
+        # Register handlers
+        self.sio.on('connect', self.on_connect)
+        self.sio.on('disconnect', self.on_disconnect)
+        self.sio.on('stream_started', self.on_stream_started)
+        self.sio.on('audio_chunk', self.on_audio_chunk)
+        self.sio.on('stream_progress', self.on_stream_progress)
+        self.sio.on('stream_complete', self.on_stream_complete)
+        self.sio.on('stream_error', self.on_stream_error)
+    
+    async def on_connect(self):
+        print(f"✅ Connected to WebSocket server")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)